index.html

```diff
@@ -205,175 +189,129 @@
           </ul>
         </dd>
         <dt>URL</dt>
         <dd>
           <p>The following terms are defined in the URL specification:
           [[!URL]]</p>
           <ul>
-            <li><dfn id='url-parser'><a href=
-            "http://url.spec.whatwg.org/#concept-url-parser">URL
-            parser</a></dfn></li>
-            <li><dfn id='scheme'><a href=
-            "http://url.spec.whatwg.org/#concept-url-scheme">scheme</a></dfn></li>
+            <li><dfn data-cite="URL#concept-url-parser">URL
+            parser</dfn></li>
+            <li><dfn data-cite="URL#concept-url-scheme">scheme</dfn></li>
           </ul>
         </dd>
         <dt>Web IDL</dt>
         <dd>
           <p>The IDL fragments in this specification must be interpreted as
           required for conforming IDL fragments, as described in the Web IDL
-          specification [[!WebIDL]].</p>
+          specification [[!WEBIDL]].</p>
           <p>The following terms are defined in the Web IDL specification:</p>
           <ul>
-            <li><dfn id='usvstring'><a href=
-            "https://heycam.github.io/webidl/#idl-USVString"><code>USVString</code></a></dfn>
+            <li><dfn data-cite="WEBIDL#dfn-throw">throw</dfn></li>
+            <li><dfn data-cite="WEBIDL#idl-USVString"><code>USVString</code></dfn>
             type</li>
-            <li><dfn id='typeerror'><a href=
-            "https://heycam.github.io/webidl/#exceptiondef-typeerror"><code>TypeError</code></a></dfn>
+            <li><dfn data-cite="WEBIDL#exceptiondef-typeerror"><code>TypeError</code></dfn>
             type</li>
           </ul>
         </dd>
-        <dt>XMLHttpRequest</dt>
+        <dt>Page Visibility</dt>
         <dd>
-          <p>The following term is defined in the XMLHttpRequest specification:
-          [[!XMLHttpRequest]]</p>
+          <p>The following term is defined in the Page Visibility specification:
+          [[!PAGE-VISIBILITY-2]]</p>
           <ul>
-            <li><dfn id='formdata'><a href=
-            "http://www.w3.org/TR/XMLHttpRequest/#formdata"><code>FormData</code></a></dfn>
-            interface</li>
+            <li><dfn data-cite="PAGE-VISIBILITY-2#dfn-visibilitychange"><code>visibilitychange</code></dfn></li>
+            <li><dfn data-cite="PAGE-VISIBILITY-2#dfn-unload"><code>unload</code></dfn></li>
+            <li><dfn data-cite="PAGE-VISIBILITY-2#dom-visibilitystate-hidden"><code>hidden</code></dfn></li>
+            <li><dfn data-cite="PAGE-VISIBILITY-2#dom-visibilitystate"><code>visiblityState</code></dfn></li>
           </ul>
         </dd>
       </dl>
     </section>
   </section>
-  <section id="sec-beacon">
+  <section>
     <h2>Beacon</h2>
-    <section id="sec-sendBeacon-method">
-      <h2><code>sendBeacon</code> Method</h2>
+    <section data-dfn-for="sendBeacon" data-dfn-for="sendBeacon">
```

Comment:
Should be:

```
data-dfn-for="Navigator" data-dfn-for="Navigator"
``` 

That should fix it :) 

I'll try to make the ReSpec more clear that you need to put the identifier of the Interface/dictionary/emun, etc. there - not the method.